.github/workflows/unit_tests.yaml

```diff
@@ -1,55 +1,14 @@
-name: Run tests
+name: CI
 
-on:
-  - push
-  - pull_request
+on: pull_request
```

Comment:
Could you also configure the support for triggering the workflow on pushes to the master branch as well?